code/datums/riding.dm

```diff
@@ -249,6 +253,71 @@
 					buckled_mob.pixel_x = 10
 					buckled_mob.pixel_y = 5
 
+//prototype repair vehicle
+/datum/riding/space/repair
+	keytype = null
+	vehicle_move_delay = 1
+	var/obj/machinery/repair_turret/turret = null
+	var/obj/vehicle/space/speedbike/repair/R = null
```

Comment:
datums should be modular, if you put a repair vehicle datum for a human it shouldnt fail